src/docfx/docset/Docset.cs

```diff
@@ -67,11 +72,12 @@ internal class Docset
         private readonly Lazy<RedirectionMap> _redirections;
         private readonly Lazy<LegacyTemplate> _legacyTemplate;
 
-        public Docset(Context context, string docsetPath, Config config, CommandLineOptions options)
+        public Docset(Context context, string docsetPath, Config config, CommandLineOptions options, string configFile)
```

Comment:
>Config config, CommandLineOptions options, string configFile [](start = 58, length = 60)

that seems weired when the parameters have both config and its name, if the configurartion is always load from Config.Load, why config doesn't know it's name?